arch/arm/boot/dts/overlays/Makefile

```diff
@@ -177,6 +177,7 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	ov5647.dtbo \
 	ov7251.dtbo \
 	ov9281.dtbo \
+	panel-dsi-generic.dtbo \
```

Comment:
Please follow the pattern of the other dsi and dpi panels and name the overlay vc4-kms-dsi-generic-overlay.dts.